docker-images/bundle/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.package.v1="strimzi-kafka-operator"
+LABEL operators.operatorframework.io.bundle.manifests.v1="manifests/"
+LABEL operators.operatorframework.io.bundle.metadata.v1="metadata/"
+LABEL operators.operatorframework.io.bundle.channel.default.v1="stable"
+LABEL operators.operatorframework.io.bundle.channels.v1="stable, strimzi-0.21.x"
```

Comment:
Should this be 0.22.0-snapshot or something?

Reply:
Could you read the value from the release.version file in the repo root, might be an argument for having a make target and/or build script for the whole process.